package.json

```diff
@@ -78,6 +78,7 @@
     "eslint-plugin-react-hooks": "^4.3.0",
     "husky": "^7.0.4",
     "lint-staged": "^12.3.5",
+    "postcss": "^8.4.8",
```

Comment:
I'd love to know the advantages of this plugin in our dApp?

Reply:
I believe it was a problem happening on the CI side when running the new Cypress version.

Reply:
`postcss` is a tool related with styles, and does it related with cypress?

Reply:
It's included in react-scripts. Including it as a devDep here fixes a missing module error during CI.